ops/testnet/staging/core/main.tf

```diff
@@ -385,21 +385,53 @@ module "lighthouse_sendoutboundroot_cron" {
   environment         = var.environment
   stage               = var.stage
   container_env_vars  = merge(local.lighthouse_env_vars, { LIGHTHOUSE_SERVICE = "sendoutboundroot" })
-  schedule_expression = "rate(30 minutes)"
+  schedule_expression = "rate(10 minutes)"
   memory_size         = 2048
 }
 
+module "lighthouse_web3signer" {
```

Comment:
Need to bring these op-root LH agents into prod and mainnet tf configs.
@preethamr Will do this.

Reply:
Done